README.md

```diff
@@ -201,6 +201,25 @@ In the VS Code extension (version 1.3.7+) set the `toitLanguageServer.command` s
 
 This makes the extension use the language server that was compiled in the [build step](#build-for-host-machine).
 
+### Jetbrains IntelliJ Platform
+
+---
+*NOTE*
+
+Applies to distributions like IntelliJ IDEA, GoLand, CLion, PHPStorm etc, but has not been tested with Fleet.
+
+---
+
+Through the plugin system, IntelliJ has limited support for LSP. In particular the [LSP Support](https://plugins.jetbrains.com/plugin/10209-lsp-support) plugin has been verified to work with Toit's LSP server.
+
+With the plugin installed, a server definition needs to be added, in Preferences > Languages & Frameworks > Language Server Protocol, with the following parameters:
+* `Executable`
+* Extension: `toit`
+* Path: _Path to toit executable_ i.e. `/usr/local/bin/toit`
```

Comment:
In context of this README, I think this should point to `$TOIT_SDK/bin/toitlsp` with a note that if you've built this yourself, you can find it in `build/host/sdk/bin/toitlsp`. No arguments should be necessary.

Reply:
Actually, it needs the same arguments as for the "Other IDEs": `--toitc=$TOIT_SDK/bin/toitc`.